src/TableTransforms.jl

```diff
@@ -49,6 +50,6 @@ export
   PCA, DRS, SDS,
   Sequential,
   Parallel,
-  →, ⊔
-
+  →, ⊔,
+  Levels
```

Comment:
Please add this transform to another place in the list. The list of transforms that is exported has an implicit logic to it. Sequential and Parallel should be the last transforms.